tasks/sep/base-001-MCP-L1/VALIDATION.md

```diff
@@ -0,0 +1,341 @@
+[system-config-etherscan]: https://sepolia.etherscan.io/address/0xba2492e52F45651B60B8B38d4Ea5E2390C64Ffb1
+[l1-xdm-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L3
+[l1-xdm-etherscan]: https://sepolia.etherscan.io/address/0xC34855F4De64F1840e5686e64278da901e261f20
+[l1-erc721-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L4
+[l1-erc721-etherscan]: https://sepolia.etherscan.io/address/0x21eFD066e581FA55Ef105170Cc04d74386a09190
+[portal-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L8
+[portal-etherscan]: https://sepolia.etherscan.io/address/0x49f53e41452C74589E85cA1677426Ba426459e85
+[batch-inbox-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/5ad42cbb49472a0bf164ade976426f7526ee6dfe/superchain/configs/sepolia/metal.yaml#L9
+[batch-inbox-etherscan]: https://sepolia.etherscan.io/address/0x24567B64a86A4c966655fba6502a93dFb701E316
+[l1-standard-bridge-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L5
+[l1-standard-bridge-etherscan]: https://sepolia.etherscan.io/address/0xfd0Bf71F60660E2f608ed56e1659C450eB113120
+[factory-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L7
+[factory-etherscan]: https://sepolia.etherscan.io/address/0xb1efB9650aD6d0CC1ed3Ac4a0B7f1D5732696D37
+[output-oracle-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L6
+[output-oracle-etherscan]: https://sepolia.etherscan.io/address/0x84457ca9D0163FbC4bbfe4Dfbb20ba46e48DF254
+
+todo: this looks like an initialization but it wasn't present in the template
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000002` <br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
+  **Meaning:** 
+
+- **Key:** `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc` <br/>
+  **Before:** `0x0000000000000000000000001126e5afb588d39c3c5465a15af389146d309581` <br/>
+  **After:** `0x000000000000000000000000ba2492e52f45651b60b8b38d4ea5e2390c64ffb1` <br/>
+  **Meaning:** Implementation address is set to the new `SystemConfig` per the [Superchain Registry][system-config-registry] and [Etherscan][system-config-etherscan].
+
+todo: this one has 12 events vs 5 events in the template, but it should be fine
```

Comment:
Interesting, what are the different events?

Reply:
These are it, seem like they were messing around with different values

https://sepolia.etherscan.io/address/0xf272670eb55e895584501d564AfEB048bEd26194#events